### DISCUSSION TITLE
toggle or hide search?

### DISCUSSION THREAD

**Discussion by unusualevent** [Pagefind/pagefind#360] [Q&A]
How do you recommend people toggle or hide the rather large search div?

**Comment by bglw** [comment]
Sorry for missing this — for integrating deeper into the search components I recommend the upcoming [modular-ui](https://www.npmjs.com/package/@pagefind/modular-ui) frontend, which is usable in its current state. I don't have any specific advice, but the modular ui package lets you hook much deeper into custom DOM elements, so the ball is more in your court for how to interact with it 🙂

Let me know if you need any more specific pointers within that
